telemetry/roles/timescaledb/tasks/initialize_db.yml

```diff
@@ -14,17 +14,23 @@
 ---
 
 
-- name: Wait for timescale pod to come to ready state
-  ansible.builtin.command: kubectl wait --for=condition=ready --timeout=10m -n "{{ namespace }}" pod -l app="{{ timescaledb_k8s_name }}"
-  changed_when: false
+- name: Wait for mysqldb pod to come to ready state
+  block:
+    - name: Wait for timescale pod to come to ready state
+      ansible.builtin.command: kubectl wait --for=condition=ready --timeout=10m -n "{{ namespace }}" pod -l app="{{ timescaledb_k8s_name }}"
```

Comment:
What about the use of `pod_timeout` variable?. I still think that if you use the Ansible K8s module you can improve this command using exponential back-off.

Reply:
As discussed, omnia will support using k8s collections instead of commands in next release